tools/tokenserver/database.py

```diff
@@ -655,13 +659,13 @@ def get_best_node(self):
         """Returns the 'least loaded' node currently available, increments the
         active count on that node, and decrements the slots currently available
         """
+        # The spanner node is the best node.
+        if self.spanner_node:
+            return self.spanner_node_id, self.spanner_node
+        # if, for whatever reason, we haven't gotten the spanner node yet...
```

Comment:
There's either no `spanner_node_id` specified or `get_spanner_node` raised an exception in the constructor so attempting to get it again here isn't necessary